### PR TITLE
vroid-studio: Add version 0.11.3

### PR DESCRIPTION
**Pull request by AndreasMReumschuessel**
I added VRoid Studio as the portable zip version.
It's a free 3D Tool to create Avatars made by the company pixiv.
Website: [https://vroid.com/en/studio/](https://vroid.com/en/studio/)

### CODE REVIEW

File: bucket/vroid-studio.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": "0.11.3",
+    "description": "A 3D character maker that lets you use your imagination to easily create your own original characters.",
+    "homepage": "https://vroid.com/en/studio",
```
Comment (Ash258) [inline bucket/vroid-studio.json:4]:
```suggestion
    "description": "A 3D character maker",
    "homepage": "https://vroid.com/en/studio/",
```

File: bucket/vroid-studio.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        "64bit": {
+            "url": "https://download.vroid.com/dist/VRoidStudio-v0.11.3-win.zip",
+            "hash": "ff41ff00954ab7159412d8369b83dfbba617f96fb2b84a783b9254e1a4213cae",
+            "bin": "VRoidStudio.exe",
+            "shortcuts": [
+                [
+                    "VRoidStudio.exe",
+                    "VRoid Studio"
+                ]
+            ]
```
Comment (Ash258) [inline bucket/vroid-studio.json:19]:
```suggestion
            "hash": "ff41ff00954ab7159412d8369b83dfbba617f96fb2b84a783b9254e1a4213cae"
```

File: bucket/vroid-studio.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        }
+    },
+    "extract_dir": "VRoidStudio-v0.11.3-win",
+    "checkver": "dist/VRoidStudio-v([\\d.]+)-win.zip",
```
Comment (Ash258) [inline bucket/vroid-studio.json:23]: Adding shim does not make sense. There is no command line support

```suggestion
    "shortcuts": [
        [
            "VRoidStudio.exe",
            "VRoid Studio"
        ]
    ],
    "checkver": "dist/VRoidStudio-v([\\d.]+)-win\\.zip",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/vroid-studio.json (hunk 3 above)
Comment (AndreasMReumschuessel) [reply bucket/vroid-studio.json:22]: True, I removed the "bin" attribute.

File: bucket/vroid-studio.json (hunk 2 above)
Comment (AndreasMReumschuessel) [reply bucket/vroid-studio.json]: I only removed the "bin" attribute. I left the shortcut there to have it searchable via the start menu of Windows.

File: bucket/vroid-studio.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
```
Comment (Ash258) [inline bucket/vroid-studio.json:22]:
```suggestion
            "hash": "ff41ff00954ab7159412d8369b83dfbba617f96fb2b84a783b9254e1a4213cae"
        }
    },
    "extract_dir": "VRoidStudio-v0.11.3-win",
    "shortcuts": [
        [
            "VRoidStudio.exe",
            "VRoid Studio"
        ]
    ],
    "checkver": "dist/VRoidStudio-v([\\d.]+)-win\\.zip",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/vroid-studio.json (hunk 4 above)
Comment (AndreasMReumschuessel) [reply bucket/vroid-studio.json]: Ok, got you.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**